goose3/crawler.py

```diff
@@ -258,8 +259,19 @@ def get_html(self, crawl_candidate, parsing_candidate):
             return crawl_candidate.raw_html
 
         # fetch HTML
-        html = self.fetcher.fetch(parsing_candidate.url)
-        return html
+        response = self.fetcher.fetch_obj(parsing_candidate.url)
+        if response.encoding != 'ISO-8859-1':  # shows that we don't actually know
+            # return response as a unicode string
+            html = response.text
+        else:
+            html = response.content
+            if 'charset' not in response.headers.get('content-type'):
+                encodings = get_encodings_from_content(response.text)
```

Comment:
I think this block should run for `raw_html` as well.

Reply:
For `raw_html` we don't have access to the request response that has an educated encoding of the html file. We do use the charset version most of the time as this is just a fallback option.

Reply:
Yes, the issue appears to be resolved. Thanks!
 
Many of the other Unicode pages didn't throw this error, but I'm guessing that's because they weren't using anything outside of the Latin-1 character set. Is that correct?